boxsdk/exception.py

```diff
@@ -193,12 +193,28 @@ def __init__(self, status, message=None, url=None, method=None):
             The HTTP verb used to make the request.
         :type method:
             `unicode`
+        :param network_response:
+            The network response for the request.
+        :type network_response:
+            :class:`NetworkResponse`
         """
         super(BoxOAuthException, self).__init__()
         self._status = status
         self._message = message
         self._url = url
         self._method = method
+        self._network_response = network_response
+
+    @property
+    def network_response(self):
+        """The response for the request.
+
+        :return network_response:
+            The network response for the request.
```

Comment:
This is exactly the same as what is said one line above. I don't think you need `:return:` for this.